### PR TITLE
build(validate-icons-colors): fix script to catch multiple fill/stroke usages

### PR DESCRIPTION
**Pull request by YossiSaadi**
https://monday.monday.com/boards/3532714909/pulses/5115330866

### CODE REVIEW

File: scripts/icons/utils.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+function validateColors(colors, svgFileName, allowedColors = ALLOWED_COLORS) {
+  colors.forEach(({ attribute, color }) => {
+    if (!allowedColors.includes(color)) {
+      throw Error(generateErrorMessage(attribute, color, svgFileName));
+    }
+  });
+}
+
+function extractColorsFromSvg(content) {
+  const colorMatches = content.match(/(fill|stroke)="([\w#]+)"/g) || [];
```
Comment (SergeyRoyt) [inline scripts/icons/utils.js:18]: Let's replace here `"` with `["']` to be more precise

File: scripts/icons/utils.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  return colorMatches.map(match => {
+    const [, attribute, color] = match.match(/(fill|stroke)="([\w#]+)"/) || [];
```
Comment (SergeyRoyt) [inline scripts/icons/utils.js:20]: Let's replace here `"` with `["']` to be more precise

**Review by SergeyRoyt — APPROVED** [review]
One note to fix the rest is great!

File: scripts/icons/utils.js (hunk 1 above)
Comment (YossiSaadi) [reply scripts/icons/utils.js]: Nice, correct, thanks!

**Pull request merged**